tests/bsim/bluetooth/audio/overlay-nrf5340_cpunet_iso-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,7 @@
+# In theory, CONFIG_BT_ISO_TX_BUF_COUNT=1, should be sufficient but this count
+# is used in the context of IPC which falls into a "Newton's Cradle" effect
+# where probably (CONFIG_BT_CTLR_ISO_TX_BUFFERS - CONFIG_BT_ISO_TX_BUF_COUNT)
+# buffers get throttled. Hence, always have the value equal or greater.
```

Comment:
Suggest to create the GH issue and reference it here rather than having this comment all the places :) Then it's easier to find and cleanup later

Reply:
Still on my todo list to create a IPC test!... 

Just not to loose track created this: #81866